development-guide.md

```diff
@@ -125,6 +125,10 @@ This operation should report all columns in the destination table, including Fiv
 - This operation might be requested for a table that does not exist in the destination. In that case, it should NOT fail, simply ignore the request and return `success = true`.
 - `utc_delete_before` has millisecond precision.
 
+#### AlterTable
+- This operation should be used for changing primary key columns, adding columns, and changing data types. 
+- However, this operation should not drop any columns even if the `AlterTable` request has a table with a different set of columns. Dropping columns could lead to unexpected customer data loss and is against Fivetran's general approach to data movement.
```

Comment:
This doesn't say what they should do. i'd also love to link to a doc that describes our policy/what we usually do - ie not dropping columns/deleting data - but I didn't just find a suitable link.

eg 'this operation should not drop any columns even if the 'AlterTabel' request has a table with a different set of columns. Fivetran's approach to data movement is to leave such columns in a customer's warehouse with no additional data populating to provide the customer control and choice on timing for removal. Dropping columns can lead to unexpected customer data loss.